### DISCUSSION TITLE
Personal Access Tokens should move; text needs to change

### DISCUSSION THREAD

**Discussion by jcairney** [community/community#7474] [Other Feature Feedback, Questions, & Ideas]
The first time I went to push a repo to my account I discovered that since August 2020 users have needed to create a Personal Access Token to push using HTTPS, since your standard account password is no longer accepted for security reasons.

So I went looking under Settings for the Personal Access Tokens setting/page, and found it under Developer Settings, but then immediately assumed this was the wrong area, because the text read:

> Need an API token for scripts or testing? Generate a personal access token for quick access to the GitHub API.

This was not the purpose of my visit, so I left and kept looking.  As it turns out, that had been the correct area, and I eventually was successful in pushing my new repo.  But I would suggest two things:

1. Personal Access Tokens should no longer be under Developer Settings.  I would posit that the majority of users now going in there and creating access tokens are not those writing extensions or plugins using the GitHub API.
2. The text should change, and though the mentions of the GitHub API should be kept, the first sentence should indicate that you are in the right place to create a Personal Access Token for use with basic Git functionality for GitHub, when using HTTPS.

Thanks for your time.  Discuss.

**Comment by alex-bradley** [comment]
👍  

Came here to say the same. Whenever my access token expires I can never remember to look under "Dev Settings".

Another thing to add. When my auth token expires, why not just tell me the reason from the CLI so I don't waste time putting in my username/password?

**Reply by Savash1996** [reply]
Seve neme...ok
